Nightwatch/Nightwatch.fsproj

```diff
@@ -19,6 +19,8 @@
     <PackageReference Include="Argu" Version="3.7.0" />
     <PackageReference Include="FSharp.Control.FusionTasks.FS41" Version="1.1.0" />
     <PackageReference Include="Quartz" Version="3.0.0-alpha3" />
+    <PackageReference Include="Serilog" Version="2.5.0" />
+    <PackageReference Include="Serilog.Sinks.Literate" Version="3.0.0" />
```

Comment:
[This package is being retired](https://github.com/serilog/serilog-sinks-literate#this-package-is-being-retired), so let's switch to console sink?